LoadingGif/LoadingGif/LoadingViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  LoadingViewController.swift
+//  LoadingGif
+//
+//  Created by Bradley Yin on 9/18/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public class LoadingViewController: UIViewController {
+    
+    private var loadingView: IndeterminateLoadingView!
+    private var button1: UIButton!
+    private var button2: UIButton!
+    
+    override public func viewDidLoad() {
+        self.view.backgroundColor = .white
+        setupUI()
+        
+        startAnimation()
+    }
+    
+    private func setupUI() {
+        let screenWidth = UIScreen.main.bounds.width
+        let loadingView = IndeterminateLoadingView(frame: CGRect(x: screenWidth / 2 - 150, y: 20, width: 300, height: 300))
+        self.view.addSubview(loadingView)
+        //loadingView.translatesAutoresizingMaskIntoConstraints = false
+        //loadingView.widthAnchor.constraint(equalToConstant: 414).isActive = true
+        //loadingView.heightAnchor.constraint(equalTo: loadingView.widthAnchor).isActive = true
+        //loadingView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 100).isActive = true
+        //loadingView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        self.loadingView = loadingView
+        
+        let button1 = UIButton()
```

Comment:
keep in mind that this is part of the framework. if another dev decides to put this in their app to show progress is occurring, everything you provide here will be part of their project now. That includes this button to dismiss the progress indicator (which really doesn't make sense in most cases - you usually want the progress indicator to disable user input until the task is finished, at which point it would automatically disappear).

the way to fix this would be to make the button part of the demo app or have the demo app just show the indicator for a couple seconds and then automatically dismiss it (simulating running a task)